### PR TITLE
Enable sub-directories sidebar

### PR DESCRIPTION
**Pull request by marktopper**
Fix for #1

### CODE REVIEW

File: src/Docs/DocsBuilder.php [hunk 1]
```diff
@@ -128,6 +129,21 @@ protected function compileWithBlade(string $content, array $data): string
         return Markdown::parse($content);
     }
 
+    protected function getCompiledDocIndex(SplFileInfo $doc, string $fallback, array $data): string
+    {
+        $path = realpath($this->docsDirectory.'/'.$doc->getRelativePath());
```
Comment (znck) [inline src/Docs/DocsBuilder.php]: Replace `'/'`  with `DIRECTORY_SEPARATOR` (To be fair with windows)

**Review by znck — CHANGES_REQUESTED** [review]

File: src/Docs/DocsBuilder.php (hunk 1 above)
Comment (marktopper) [reply src/Docs/DocsBuilder.php]: I have now changed that line.

**Review by znck — APPROVED** [review]

**Comment by znck** [comment]
Thanks ✌️

**Comment by marktopper** [comment]
My pleasure ✌️

**Pull request merged**